File: 0009 palindrome-number/solution.cs
public class Solution {
    public bool IsPalindrome(int x) {
        int o=x,rev=0;
        while(o>0) {
            int dig=o%10;
            rev=rev*10+dig;
            o/=10;
        }
        return rev==x;
    }
}

File: 0009 palindrome-number/solution.rs
impl Solution {
    pub fn is_palindrome(x: i32) -> bool {
        let mut o=x;
        let mut r=0;
        while o>0 {
            let dig=o%10;
            r=r*10+dig;
            o/=10;
        }
        x==r
    }
}
